modules/ROOT/examples/live-demos/revisionhistory/index.js

```diff
@@ -183,17 +183,17 @@ const revisions = [
 ];
 
 const revisionhistory_fetch_revision = (_editor, revision) =>
-  new Promise((resolve) => {
+  new Promise((resolve, reject) => {
     setTimeout(() => {
-      let newRevision = null;
       for (let i = 0; i < revisions.length; i++) {
```

Comment:
```suggestion
      const newRevision = revisions.find((r) => r.revisionId === revision.revisionId);
      if (newRevision === undefined) {
    	reject(`Revision ${revision.revisionId} is not found`);
      } 
      resolve(newRevision);
```
Just a suggestion 👍

Reply:
This is acceptable because IE is no longer supported :)